.github/workflows/benchmark-rust.yml

```diff
@@ -63,3 +63,79 @@ jobs:
         with:
           run: cargo codspeed run -p bench
           token: ${{ secrets.CODSPEED_TOKEN }}
+
+  rust-benchmark:
+    name: Rust Benchmark
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v4
+        with:
+          # Whether Pull submodules for additional files
+          submodules: false
+
+      - name: Install Rust
+        uses: moonrepo/setup-rust@v1
+        with:
+          bins: just
+          cache-base: main
+          cache-target: release
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Setup benchmark
+        run: just setup-bench
+
+      - name: Download previous benchmark data
+        uses: actions/checkout@v4
+        with:
+          # https://github.com/rolldown/benchmark-results-storage
+          repository: rolldown/benchmark-results-storage
+          path: tmp
+
+      - name: Run the benchmarks
+        run: cargo bench -p bench | tee tmp/new-benchmark-rust-output.txt
+
+      - name: Show/Update benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: 'Rust Benchmark'
+          # What benchmark tool the output.txt came from
+          tool: 'cargo'
+          # Where the output from the benchmark tool is stored
+          output-file-path: tmp/new-benchmark-rust-output.txt
+          # Where the previous data file is stored
+          external-data-json-path: tmp/benchmark-rust-output.txt
```

Comment:
I was hoping we are doing this way: https://github.com/rolldown/rolldown/blob/6f01071b31f5255ce0a0427a3a67c4757f96504d/.github/workflows/bench.yaml#L37-L59

Using previous data is most likely inaccurate due to they are running in different github machines.

We could just use github-benchmark-action to store data and draw chart.

Reply:
> From experences of nodejs benchmark CI, using previous data is most likely inaccurate due to they are running in different github machines.

Yeah. We need to run benchmark at one machines. The store data and draw chart is unnecessary, because it has some noise.